templates/common/_base/units/machine-config-daemon-pull.service.yaml

```diff
@@ -15,7 +15,7 @@ contents: |
   [Service]
   Type=oneshot
   RemainAfterExit=yes
-  ExecStart=/bin/sh -c "while ! /usr/bin/podman pull --authfile=/var/lib/kubelet/config.json '{{ .Images.machineConfigOperator }}'; do sleep 1; done"
+  ExecStart=/bin/sh -c "while ! /usr/bin/podman pull --signature-policy /etc/machine-config-daemon/policy-for-old-podman.json --authfile=/var/lib/kubelet/config.json '{{ .Images.machineConfigOperator }}'; do sleep 1; done"
```

Comment:
I think this should work, and I'm fine for this as a stopgap, but I think we should either:
1. add conditionals to check the podman version here and only use the fallback for old podman if we're too old, or
2. add a followup jira card and comment here to remove this once we have some OS bootimage version skew policy in place.

If we can do 1 then new users can still benefit from the updated policy (albeit just for one image pull), WDYT?

Reply:
I agree with the two points you mentioned. For now, just use this WIP version for testing, will double-check if this can work :crossed_fingers: